lib/modules/filteReddit.js

```diff
@@ -586,6 +586,23 @@ addModule('filteReddit', function(module, moduleID) {
 						return thing.classList.contains('over18');
 					}
 				},
+				loggedInAs: {
+					name: 'When logged in as a certain user',
+					defaultTemplate: function(patt) {
+						return {type: 'loggedInAs', patt: patt || ''};
+					},
+					fields: [
+						'logged in as /u/', {type: 'text', id: 'loggedInAs', validator: RegExp}
+					],
+					evaluate: function(thing, data, config) {
+						var myName = RESUtils.loggedInUser();
+						if (!myName) {
+							return false;
+						}
+						var pattern = new RegExp('^('+data.loggedInAs+')$', 'i');
```

Comment:
Yes, as in https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/master/lib/modules/filteReddit.js#L381
Looking at http://stackoverflow.com/q/2140627, RegExp actually seems to be one of the less ugly methods for case-insensitive string comparison (when escaping control characters). It still breaks for languages such as turkish, where uppercase i is İ and lowercase I is ı - see https://jsfiddle.net/csu3pf9a/1/. Actual language-independent case-insensitive string comparison seems to be impossible.